index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Porfolio Website</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
+
+        <!--Header-->
+        <header>
+            <a href="#">John Doe </a>
+
+            <nav id="navig">
+            <button id="hum">&#9776;</button>
+                <div id="navitems">
+                    <ul>
+                      <li><a href="#">Portfolio</a></li>
+                      <li><a href="#">About</a></li>
+                      <li><a href="#">Contact</a></li>
+                      <li><a href="#"><img alt="Mail" src="/assets/mail-icon.svg" width="20" height="30"></a></li>
+                    </ul>
+                </div>
+            </nav>
+</header>
+<!--Toolbar-->
+<section id="toolbar">
+    <div id="heytext">
+        <h1>Hey There.<br>
+            I’m Alex </h1>
+    </div>
+    <div id="whoami">
+        <p id="p3">I’am a Software Developer<p>
```

Comment:
- [ ] As the previous reviewer mentioned, I highly recommend using the `hx` tag (h2,h3,h4,...) while this is a sub-title, it is better to use heading tags. :wink: